lib/macos.js

```diff
@@ -19,9 +19,29 @@ const parseMac = stdout => {
 	}
 };
 
-module.exports = async () => {
-	const {stdout} = await execFile(bin);
+const getArguments = options => {
+	if (!options) {
+		return [];
+	}
+
+	const args = [];
+	if (options.accessCheck === false) {
```

Comment:
I think screen recording and accessibility checks should be separate options and more specifically named.